chapter2/tests/t19/terraform/dev/dev.tfvars

```diff
@@ -0,0 +1,39 @@
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ env=specific configuration variables                                                                             │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+environment = "dev"
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ nginx configuration variables                                                                                    │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+nginx = {
+  image          = "nginx:1.21"
+  container_name = "saritasa-devops-camps-2023-workspace-nginx-dev"
+  container_ports = {
+    internal = 80,
+    external = 8080
+  }
+  container_volumes = [
+    {
+      # host - in root project folder
```

Comment:
comments shows be in the inline box above `nginx configuration variables` not here.

Reply:
and explain that you construct full path somewhere else

Reply:
> comments shows be in the inline box above `nginx configuration variables` not here.

@dmitry-mightydevops 
Done

Reply:
> and explain that you construct full path somewhere else

@dmitry-mightydevops 
Done